openedx/core/djangoapps/waffle_utils/__init__.py

```diff
@@ -360,8 +354,9 @@ def __init__(self, waffle_namespace, flag_name, flag_undefined_default=None):
         Arguments:
             waffle_namespace (WaffleFlagNamespace | String): Namespace for this flag.
             flag_name (String): The name of the flag (without namespacing).
-            flag_undefined_default (Boolean): A default value to be returned if
-                the waffle flag is to be checked, but doesn't exist.
+            DEPRECATED flag_undefined_default (Boolean): A default value to be returned
```

Comment:
Can we also log a deprecation warning in the code when this parameter is not none?

Reply:
@feanil: I will be removing this in the next few days, so don't think it is needed.  Hope that is fine.